.env.development

```diff
@@ -0,0 +1,8 @@
+#ULOAD-REQUEST
+VITE_UPDATE_UPLOAD_REQUEST_TIME=6000
+
+#LINK-GEY-UPLOAD-REQUEST
+VITE_LINK_GET_UPLOAD_REQUEST="https://gx9mczg2il.execute-api.us-east-1.amazonaws.com/api/v1/users/1/upload/requests"
+
+#LINK-POST-INSERT-REQUEST
```

Comment:
Do we really need this comment? I understand the var name even without a comment :)